docs/src/content/docs/es/guides/sidebar.mdx

```diff
@@ -32,20 +32,15 @@ La siguiente barra lateral se generará automáticamente:
 <SidebarPreview
 	config={[
 		{
-			label: 'guides',
+			label: 'constellations',
 			items: [
-				{ label: 'Componentes', link: '/guides/components/' },
-				{ label: 'Internacionalización (i18n)', link: '/guides/i18n/' },
+				{ label: 'Andromeda', link: '' },
+				{ label: 'Orion', link: '' },				
```

Comment:
In other translations, so far we have been trying to translate the example, e.g. ["Andrómeda"](https://es.wikipedia.org/wiki/Andr%C3%B3meda_(constelaci%C3%B3n)) for "Andromeda", ["Orión"](https://es.wikipedia.org/wiki/Ori%C3%B3n_(constelaci%C3%B3n)) for "Orion", etc.

Would it make sense to do the same in Spanish too? If yes, I guess this would apply to all examples in this page. If not, do not hesitate to tell me if for example it's a domain where translation are not common, etc.